### DISCUSSION TITLE
Error with TimeSeriesKMeans class import

### DISCUSSION THREAD

**Discussion by foe12** [alan-turing-institute/sktime#2949] [General]
Please, help me to understand why I cant to import class TimeSeriesKMeans?  All my efforts to import TimeSeriesKMeans are handling to infinity. 
For clarity, I have jupyter notebook with 1 row: 
`from sktime.clustering.k_means import TimeSeriesKMeans`
sktime version is 0.11.2
OS windows

When I interrupt kernel I see errors:
[err_log.txt](https://github.com/alan-turing-institute/sktime/files/9062183/err_log.txt)

Hope for your help to understand what's wrong. Thank you in advance!

**Comment by fkiraly** [comment]
Hm, doesn't look related to `sktime`, could be file permissions in your local python cache, where a file might be locked?
Maybe a restart of your computer might help, and/or deleting the folders showing up?

I ran the same line for import and have no problems.
